packages/api/src/middleware/auth.ts

```diff
@@ -108,7 +108,8 @@ function authenticator(): RequestHandler {
       );
     }
 
-    user = await db.user.get(userId);
+    user = await db.user.get(userId, { cache: true });
```

Comment:
We got a problem here - tests are failing when checking if playback is disallowed to suspended users because the cached user is not suspended

Reply:
Also, some other tests are failing due to the User being unverified. We are relying too much to users not being cached

Reply:
Yeah we need to update the tests either not to cache at all or to wait until it's expired before checking

Reply:
Oh a better option should be a test function to clear the cache

Reply:
done